support-frontend/typescript-errors.json

```diff
@@ -107,6 +104,8 @@
             "assets/helpers/polyfills/layout.ts": 1,
             "assets/pages/digital-subscription-checkout/components/endSummary/endSummarySelector.ts": 1,
             "assets/pages/paper-subscription-checkout/helpers/orderSummaryText.ts": 1,
+            "assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.tsx": 1,
+            "assets/pages/weekly-subscription-checkout/components/weeklyCheckoutFormGifting.tsx": 1,
```

Comment:
I think these two new ones are because `datalist` is required in the `SummaryPropTypes`. I'm not quite sure why this wasn't an issue before.

I think it's quite a simple fix though, as we've already got `[]` in place as a fallback for when they're not passed so I think we can just mark `datalist` as optional?

Reply:
Ah nice spot, fix incoming...